microsoft/vscode
# Allow interactive window to scroll beyond end (#127578)

This PR fixes #127578

To test:

1. Install a notebook extension that implements an interactive window.
2. Open an interactive window and run some commands.
3. Verify that the scrolling region above the interactive input field can be scrolled to leave blank lines between the last output cell and the interactive input field.

/assign @rebornix

ping @rchiodo
